screenpy/__init__.py

```diff
@@ -19,31 +19,72 @@
 :license: MIT, see LICENSE for more details.
 """
 
+from . import actions, narration, resolutions
+from .actions import *  # noqa
```

Comment:
I had put about four comments on lines like these asking if they were necessary now that we add the `__all__` bits together, then i realized that just adding them to `__all__` does not actually import them. :P